src/Turbo/CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # CHANGELOG
+## 2.22.0
+
+-   Add support for custom actions in `TurboStream` and `TurboStreamResponse`
```

Comment:
Arf, saw this just too late... can you move it a couple lines down, there was already a 2.22.0 block 😅